client/checkout/blocks/payment-elements.js

```diff
@@ -77,6 +77,13 @@ const PaymentElements = ( { api, ...props } ) => {
 					paymentMethodTypes: paymentMethodTypes,
 					appearance: appearance,
 					fonts: fontRules,
+					...( paymentMethodTypes.includes( 'wechat_pay' ) && {
+						paymentMethodOptions: {
+							wechatPay: {
+								client: 'web',
+							},
+						},
+					} ),
```

Comment:
It looks like this isn't necessary, since the server is sending this option directly when creating the intent.

Reply:
I wondered about this but forgot to remove check. Thanks!